test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,70 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+public class FindCommandTest {
+
+    private AddressBook addressBook;
+    private TypicalPersons td;
```

Comment:
Can make these final as well.

Actually, I think you can just assign the fields here and do without the `setUp()` method (JUnit constructs a new instance for every test).